meeshkan/serve/mock/matcher.py

```diff
@@ -339,6 +339,29 @@ def change_refs(j: Schema) -> Schema:
             change_ref(item) if isinstance(item, Reference) else change_refs(item)
             for item in j.items
         ],
+        _not=None
+        if j._not is None
```

Comment:
Interesting that black does not indent this, it's a bit surprising formatting 🤔

Reply:
Agreed